boards/common/blxxxpill/Makefile.include

```diff
@@ -22,6 +22,15 @@ ifeq (dfu-util,$(PROGRAMMER))
   ROM_OFFSET ?= 0x2000
 endif
 
+# The boards debug header only exports  SWD, so JTAG-only pins PA15, PB3(*),
+# and PB4 can be remapped as regular GPIOs instead.
+#
+# Note: PB3 is also used as SWO. The user needs to take care to not enable SWO
+# with the debugger while at the same time PB3 is used as GPIO. But RIOT does
+# not use SWO in any case, so if a user adds this feature in her/his own code,
+# she/he should be well aware of this.
+CFLAGS += -DCONFIG_AFIO_MAPR_SWJ_CFG=SWJ_CFG_NO_JTAG_DP
```

Comment:
If we are not using SWO, maybe this should be the default?

Reply:
Indeed, I think we are not even able to flash via JTAG in the first place